content/install-c4.md

```diff
@@ -0,0 +1,233 @@
+# Install the Wazo Platform Class 4
+
+## Supported environments
+
+Wazo Platform Class 4 supports the following deployment environments:
+
+- **Ansible-based deployment**: install the platform on a set of Debian Buster machines, either bare-metal or virtual machines, using our Ansible recipes.
+- **Docker compose**: run the platform on a single machine using the Docker images together with docker-compose.
+- **Kubernetes**: install the platform using our Helm chart and run the platform on your Kubernetes cluster.
+
+## Ansible-based deployment
+
+The Wazo Platform Class 4 is made of several components:
+
+1. SBC (Kamailio)
+2. Router (Kamailio)
+3. Media Proxy (RTPengine)
+4. Router API (`wazo-router-confd`)
+5. PostgreSQL
+6. Redis
+
+If you use the Ansible recipes you need at least two hosts as you cannot run the SBC and the Router on the same machine.
+
+To install the Wazo Platform Class 4 in an all-in-one setup, do the following steps:
+
+1. Install a Debian 10 Buster system with a default locale with an UTF-8 charset on each host you want to use, either bare metal or virtual machine.
+
+2. Run the following commands as root on the Debian systems to provision sudo, git and Ansible:
+
+```ShellSession
+# apt-get install -yq sudo git ansible
+```
+
+3. Get the Wazo Platform installer and its dependencies:
+
+```ShellSession
+# git clone https://github.com/wazo-platform/wazo-ansible.git
+# cd wazo-ansible
+# ansible-galaxy install -r requirements-postgresql.yml
+```
+
+4. Create the Ansible inventory in `inventories/c4` to specify your hosts and your preferences, as follows:
+
+Define one or more SBCs:
+
+```Ini
+[sbc_host]
+sbc_1 ansible_ssh_host=10.0.1.1 ansible_ssh_port=22
```

Comment:
It is better to use IP addresses reserved for documentation : https://tools.ietf.org/html/rfc5737 
blocks 192.0.2.0/24 (TEST-NET-1), 198.51.100.0/24 (TEST-NET-2), and 203.0.113.0/24 (TEST-NET-3)

Reply:
@mwolff44 ok